websockets/README.md

```diff
@@ -24,19 +24,22 @@ This object MUST NOT contain any properties. Its name is reserved for future use
 
 When using WebSockets, the channel represents the connection. Unlike other protocols that support multiple virtual channels (topics, routing keys, etc.) per connection, WebSockets doesn't support virtual channels or, put it another way, there's only one channel and its characteristics are strongly related to the protocol used for the handshake, i.e., HTTP.
 
-##### Fixed Fields
+#### Fixed Fields
 
 Field Name | Type | Description
 ---|:---:|---
 <a name="operationBindingObjectMethod"></a>`method` | string | The HTTP method to use when establishing the connection. Its value MUST be either `GET` or `POST`.
 <a name="operationBindingObjectQuery"></a>`query` | [Schema Object][schemaObject] | A Schema object containing the definitions for each query parameter. This schema MUST be of type `object` and have a `properties` key.
 <a name="operationBindingObjectHeaders"></a>`headers` | [Schema Object][schemaObject] | A Schema object containing the definitions of the HTTP headers to use when establishing the connection. This schema MUST be of type `object` and have a `properties` key.
 <a name="operationBindingObjectQueryValues"></a>`queryValues`| object| An object containing query values.
-<a name="operationBindingObjectHeaderValues"></a>`headerValues`| object | An object containing header values
+<a name="operationBindingObjectHeaderValues"></a>`headerValues`| object | An object containing header values.
 <a name="operationBindingObjectBindingVersion"></a>`bindingVersion` | string | The version of this binding. If omitted, "latest" MUST be assumed.
 
 This object MUST contain only the properties defined above.
 
+#### Passing ENV variables to headers and query 
+You can pass in enviornment variables to `queryValues` and `headerValues` using `$<variable>` format. Tools will resovle this value using `.env` files. 
```

Comment:
A few comments on this:

1. Let's try to use more formal language when possible. For instance, use language like "All string values containing words starting with the dollar ($) sign, MUST be replaced with environment variables of the same name at runtime." instead of "You can pass" or "Tools will resolve".
2. `.env` files are something very typical/specific from Node.js. Also, when running in production, these values are usually retrieved from the system environment variables, not from a file.
3. Provide some examples of Do's and Dont's. For instance, `Bearer $TOKEN` would be ok but `Bearer $ TOKEN` would not.
4. Are these variables case-sensitive or not? I don't think they should be and we should probably make it mandatory to be always uppercase.
5. Can these variable names start with numbers? What symbols are allowed? You should probably check what symbols are valid for environment variables in Unix and Windows.

Reply:
I couldn't find any solid document where it says for certain rule how to name env variables, but generally https://pubs.opengroup.org/onlinepubs/000095399/basedefs/xbd_chap08.html is something we can use. We allow only uppercase, underscore, and digits but do not start with digits.